lib/src/asset.dart

```diff
@@ -92,7 +99,8 @@ class AssetGiffyDialog extends StatelessWidget {
                     color: buttonOkColor ?? Colors.green,
                     shape: RoundedRectangleBorder(
                         borderRadius: BorderRadius.circular(buttonRadius)),
-                    onPressed: onOkButtonPressed ?? () {},
+                    onPressed:
+                        onOkButtonPressed, // On passing Null button will not look clickable
```

Comment:
No need to change this part as I already handled the null part by passing an empty method.

Reply:
Passing an empty method will make the button seem clickable but on clicking it user will not see any action, however if you pass null to it, the button will be grayed out and not animate upon clicking.

Reply:
Yes, I got the point. Please fix that other change above this and try resolving the merge conflicts.